chat/command.go

```diff
@@ -173,9 +173,15 @@ func InitCommands(c *Commands) {
 		Prefix: "/names",
 		Help:   "List users who are connected.",
 		Handler: func(room *Room, msg message.CommandMsg) error {
-			// TODO: colorize
-			names := room.NamesPrefix("")
-			body := fmt.Sprintf("%d connected: %s", len(names), strings.Join(names, ", "))
+			names := room.Members.ListPrefix("")
+			colNames := make([]string, len(names))
+			theme := msg.From().Config().Theme
```

Comment:
I thiiiink it's possible for Theme to be nil.

Reply:
I'll add a check for nil and write an early exit.